### ISSUE TITLE
Add a gender option

### ISSUE THREAD

**Issue by th3c0d3br34ker** [ashutosh1919/masterPortfolio#68]
Add an option for Male/Female where the illustrations would come up accordingly.

Saw [Tanya Goel's](https://tanya-goel.github.io/myMasterPortfolio/) Porfolio and realized that there is no such option.

**Comment by Tanya-Goel** [comment]
Yeah,it seems a good option.Otherwise one has to change front picture at the home page of portfolio.

**Comment by ashutosh1919** [comment]
Also, we need to add flexibility to remove tabs.

**Comment by saiteja13427** [comment]
The development of this issue was not completed. 

So, this issue is now open for anyone to take up. Please do comment here if you want to work on it and I would assign it to you.

**Comment by GauthamRVanjre** [comment]
Hello @saiteja13427, I am ready to take this issue can you please assign it to me under Hacktoberfest 2023

**Comment by ashutosh1919** [comment]
@GauthamRVanjre please raise PR with sufficient code fix and we will assign this issue to you
